app/src/androidTest/java/org/mozilla/fenix/ui/robots/NotificationRobot.kt

```diff
@@ -54,13 +51,20 @@ class NotificationRobot {
         assertPrivateTabsNotification()
     }
 
-    fun clickMediaSystemNotificationControlButton(action: String) {
+    fun clickSystemNotificationControlButton(action: String) {
         mediaSystemNotificationButton(action).waitForExists(waitingTime)
         mediaSystemNotificationButton(action).click()
     }
 
     fun verifyMediaSystemNotificationButtonState(action: String) {
-        mediaSystemNotificationButton(action).waitForExists(waitingTime)
+        assertTrue(mediaSystemNotificationButton(action).waitForExists(waitingTime))
+    }
+
+    fun expandNotificationMessage() {
+        while (!notificationHeader.exists()) {
```

Comment:
Does this need to be API checked?

Reply:
It's the same ID on the following OS versions.